recursion_and_iteration.rb

```diff
@@ -0,0 +1,59 @@
+def factorial_r(num)
+  if num == 0
+    1
+  else
+    num * factorial_r(num - 1)
+  end
+end
+
+def factorial_i(num)
+  big_number = num
+  if num == 0
+    big_number = 1
+  else
+    while num > 1    
+      big_number = big_number * num
+      num -= 1
+    end
+  end
+  puts big_number
+end
+
+def sumdig_r(num)
+  if num < 10
+    num
+  else
+    num % 10 + schwadulo(num/10)
+  end
+end
+
+def sumdig_i(num)
+  iterator = 10
+  answer = 0
+  until num == 0
+    answer += num % 10
+    num = num / 10
+  end
+  puts answer
+end

```

Comment:
You define `iterator` but then never call it for the rest of the method.  `/=` looks crazy but it will work on line 35. Lastly, this method will `puts` an answer but return `nil`.